vm/src/builtins.rs

```diff
@@ -570,27 +570,68 @@ fn builtin_pow(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
 
 pub fn builtin_print(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
     trace!("print called with {:?}", args);
+
+    // Handle 'sep' kwarg:
+    let sep_arg = args
```

Comment:
How about the `file` argument? If we have this one, we can de-duplicate the print method in the wasm codebase.

Reply:
I started playing with an idea of a trait for `print` destinations, to implement the `file` argument, while keeping static dispatch for stdout. And yes, that would deduplicate the wasm code. But I havwn’t finishes this yet.

I want to create a wrapper trait `PyPrintDst` which behaves similar to Rust `io::Write`, but also takes `&mut VirtualMachine` as parameter in its methods and returns `PyObjectRef`s as error values.

That should make it easy to do printing to stdout, Strings, files, and custom objects implementing `write` and (optionaly for `flush=True`) `flush` methods.